src/nupic/frameworks/viz/graphviz_renderer.py

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2017, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import io
+import networkx as nx
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+
+
+
+class GraphVizRenderer(object):
+  """
+  Network visualization "renderer" implementation to render a network with
+  graphviz.
+  """
+  def render(self, G):
+    G = nx.nx_agraph.to_agraph(G)
```

Comment:
Thanks.  All of the networkx documentation and related literature tend to use the `G` convention.  I will change to `graph` per your request.